dev/com.ibm.ws.webcontainer.security_test.servlets/src/com/ibm/ws/webcontainer/security/test/servlets/ServletClientImpl.java

```diff
@@ -599,6 +600,7 @@ protected String getSSOCookie(HttpMessage httpMessage, String cookieName) {
                 }
             }
         }
+        this.cookiePath = cookiePath;
```

Comment:
```suggestion
        this.cookiePath = null;
```

If the local cookiePath value is never set to anything besides null at initialization, then we can just use null directly, and remove the local String cookiePath.